### ISSUE TITLE
MixTestInteractive.PortRunner.run incorrectly assembles command

### ISSUE THREAD

**Issue by callmiy** [randycoulman/mix_test_interactive#104]
Related to #59 and a config like:

```elixir
# config/config.exs
import Config

if Mix.env == :dev do
  config :mix_test_interactive,
    command: {"elixir", ["--sname", "name", "-S", "mix"]}
end
```

Add the following test ([may be after port_runner_test.exs#L120](https://github.com/randycoulman/mix_test_interactive/blob/cf6e09aae20b6c4893941cff6ed367d90990dfc0/test/mix_test_interactive/port_runner_test.exs#L120))

```elixir 
    test "assembles correct command with elixir --sname" do
      config = %Config{command: {"elixir", ["--sname", "my-node", "-S", "mix"]}}

      {_command, args, _options} = run_unix(config: config)

      assert args == [
        "elixir",
        "--sname",
        "my-node",
        "-S",
        "mix",
        "do",
        "run",
        "-e",
        "Application.put_env(:elixir, :ansi_enabled, true);",
        ",",
        "test"
      ]
    end
```

Test fails with:

```elixir 
test/mix_test_interactive/port_runner_test.exs:122
Assertion with == failed
code:  assert args == [
        "elixir",
        "--sname",
        "my-node",
        "-S",
        "mix",
        "do",
        "run",
        "-e",
        "Application.put_env(:elixir, :ansi_enabled, true);",
        ",",
        "test"
      ]
left:  ["elixir", "do", "run", "-e", "Application.put_env(:elixir, :ansi_enabled, true);", ",", "--sname", "my-node", "-S", "mix", "test"]
right: ["elixir", "--sname", "my-node", "-S", "mix", "do", "run", "-e", "Application.put_env(:elixir, :ansi_enabled, true);", ",", "test"]
```

I think the assumption made in `MixTestInteractive.PortRunner.run` is that custom args should be appended. But in the case of `elixir [other-args] -S mix`, `[other-args] -S mix` should be prepended.

[I made a pull request](https://github.com/randycoulman/mix_test_interactive/pull/105)

**Comment by randycoulman** [comment]
Fixed in #105

**Comment by randycoulman** [comment]
@callmiy The fix has been released as v3.2.1. Let me know if it still doesn't work for you.

**Comment by callmiy** [comment]
Works perfectly. Thanks 🙏.
